lib/src/navbar/navbar.component.ts

```diff
@@ -15,10 +16,40 @@ export class NavbarComponent implements OnInit {
 
     @Input() fixedTop: boolean = false;
 
+    @ContentChildren(NavbarMenuComponent) mobileMenu: QueryList <NavbarMenuComponent>;
+
+    menuOpen: boolean = false;
+    hamburger: string = 'inline';
```

Comment:
Not important, but just for your consideration, this might be a cleaner approach to swapping between the hamburger and close icon:

Instead of passing around css values, you could get rid of the `hamburger` and `hamburgerClose` properties entirely, and could have a `mobileMenuIcon` property like this:
```
get mobileMenuIcon(): string { return this.menuOpen ? 'fa-times' : 'fa-bars'; }
```

and then pass that into a single hc-icon in navbar.component.html like this:
```
<hc-icon fontSet="fa" [fontIcon]="mobileMenuIcon" icon-md></hc-icon>
```

Reply:
Great suggestion, very clever - and a good use of @jtrujill's hc-icon.  Although it did reveal one small bug in that component.  In the update function, it was checking whether there was a previous icon, and if so removing it.  But the previousIcon string wasn't ever getting set so the old icon wasn't being removed.  Made that change and all seems to be well.  

Thanks for the tip @corykon - this is definitely a much more elegant way to handle the icon swap.